yt_dlp/extractor/weibo.py

```diff
@@ -18,24 +18,30 @@
 
 
 class WeiboBaseIE(InfoExtractor):
-    def _update_visitor_cookies(self, video_id):
+    def _update_visitor_cookies(self, visitor_url, video_id):
+        chrome_ver = self._search_regex(
```

Comment:
can assign headers dict to a variable here as it will be used in multiple requests
```suggestion
        headers = {'Referer': visitor_url}
        chrome_ver = self._search_regex(
```